sb_vision/tokens.py

```diff
@@ -134,19 +138,32 @@ def _get_cartesian(
     homography_matrix,
     image_size: Tuple[int, int],
     distance_model: str,
+    marker_size: Tuple[float, float]=None,
```

Comment:
I'm not sure we want this argument to be optional, though we presumably need to allow it to be `None` in case there is no size. 78dd5a6aba5a846ba85b86a82461cc31ad1db2f4 changes it to explicit `Optional[...]` while requiring the argument to be present.